ixmp4/conf/logging/development.json

```
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "disable_existing_loggers": false,
+  "formatters":{
+    "generic":{ 
+      "format": "[%(levelname)s] %(asctime)s - %(name)s: %(message)s",
+      "datefmt": "%H:%M:%S"
+    }
+  },
+  "root":{ 
+    "level": "INFO",
+    "handlers": ["console"]
+  },
+  "handlers":{
+    "console":{
+      "class": "logging.StreamHandler",
+      "level": "NOTSET",
+      "formatter": "generic"
+    }
+  }
+}
```

Add empty line at end of file
```suggestion
}

```